### PR TITLE
Delete messages with ts more than 90 days old

### LINKED ISSUE #1: Clean old messages from database
- Clean old messages from database to prevent db from growing too large

### PR DESCRIPTION
**Pull request by JakeBobAnderson**
Closes https://github.com/hackgvl/slack-events-bot/issues/1

### CODE REVIEW

**Comment by oliviasculley** [comment]
This looks good, but I've recently added black as a code formatter into the project to help make things a bit more readable. Could you rebase off of `dev` and make sure that the proposed changes are formatted via black? Thank you!

**Comment by JakeBobAnderson** [comment]
@oliviasculley I'm using the bundled formatter from the Black Formatter VSCode extension with args [--line-length=100] and am having a hard time getting formatting to line up between the files. Can you share what black version and args you're using?

**Comment by oliviasculley** [comment]
As far as I'm aware, we're using the default settings for everything, including the column width at 88 characters

File: src/database.py [hunk 1]
```diff
@@ -7,26 +7,26 @@ def create_tables(conn):
 
     cur.executescript(
         """
-		CREATE TABLE IF NOT EXISTS channels (
-			id integer PRIMARY KEY AUTOINCREMENT NOT NULL,
-			slack_channel_id TEXT UNIQUE NOT NULL
-		);
+        CREATE TABLE IF NOT EXISTS channels (
```
Comment (JakeBobAnderson) [inline src/database.py]: My IDE was complaining about spaces vs. tabs so I just converted these to tabs

**Comment by oliviasculley** [comment]
It looks like this has passed the linter, so if you're ready to merge it in, we can unmark it as a draft PR and I can change the PR requirements from needing `lint_and_test` to pass, which is what we're wanting to change it in another PR that isn't as ready as this one I think

**Comment by JakeBobAnderson** [comment]
I just unmarked this from draft status and caught it back up to the dev branch. Looks like it's seen a lot of changes recently so it might be easier to wait until the hack greenville meetup to test and merge it in. Or we can merge it in now, whatever you want to do 👍

**Comment by allella** [comment]
Deferring to Olivia for a review, but wanted to say "thanks for contributing".

**Comment by oliviasculley** [comment]
@all-contributors please add @JakeBobAnderson for code!

**Comment by oliviasculley** [comment]
This looks good, but it seems like there's just a couple of pylint issues with redefines/reimports that should be easy to fix, but it should be ready after that! Thank you!

**Comment by oliviasculley** [comment]
Hey, there were a bit of churn with imports as well, could you rebase off of `dev` which adds isort and ssort when you fix the pylint warnings? Thank you!

File: src/server.py [hunk 2]
```diff
@@ -175,7 +188,7 @@ async def health_check(req: Request):
         thread.daemon = True
         thread.start()
     except (KeyboardInterrupt, SystemExit):
-        thread.join()
+        thread.join(timeout=60)
```
Comment (JakeBobAnderson) [inline src/server.py:191]: I think if we don't have a timeouts on these joins they'll block until the sleep calls finish -- meaning we could be running up to 24 hours post termination waiting on the joins. How do you feel about this change? I'm also thinking we can maybe in a future PR use mutexes to know if database operations are happening and when to terminate.

**Comment by JakeBobAnderson** [comment]
@oliviasculley Thanks for your patience on this one! I think it is ready for re-review now.

File: src/server.py (hunk 2 above)
Comment (oliviasculley) [reply src/server.py:191]: This is a great change, thank you!

**Review by oliviasculley — APPROVED** [review]

**Comment by oliviasculley** [comment]
Everything looks good but unfortunately I can't merge it in :sob:  :sob:

Could you try doing `git rebase -i origin/dev`? It could probably all be squashed to one commit at the same time, interactive rebases are cool like that

**Comment by JakeBobAnderson** [comment]
Oh no! 😂 I'm out of the office, but I'll rebase and force push it when I get home.

**Pull request merged**